features/org.wso2.carbon.siddhi.extensions.installer.core.feature/src/main/resources/extensionsInstaller/extensionDependencies.json

```diff
@@ -319,6 +319,181 @@
+        },
+        "usages": [
+          {
+            "type": "BUNDLE",
+            "usedBy": "RUNTIME"
+          }
+        ],
+        "lookupRegex": "siddhi([_-])io([_-])cdc([_-])2.0.3.jar"
+      },
+      {
+        "name": "postgresql",
+        "version": "42.2.10",
+        "download": {
+          "autoDownloadable": true,
+          "url": "https://repo1.maven.org/maven2/org/postgresql/postgresql/42.2.10/postgresql-42.2.10.jar"
+        },
+        "usages": [
+          {
+            "type": "BUNDLE",
+            "usedBy": "RUNTIME"
+          }
+        ],
+        "lookupRegex": "postgresql([_-])42.2.10(.+).jar"
+      }
+    ]
+  },
+
+  "cdc-mssql": {
+    "extension": {
+      "name": "cdc-mssql",
+      "displayName": "Change Data Capture - MS SQL",
+      "version": "2.0.3"
+    },
+    "dependencies": [
+      {
+        "name": "siddhi-io-cdc",
+        "version": "2.0.3",
+        "download": {
+          "autoDownloadable": true,
+          "url": "https://repo1.maven.org/maven2/io/siddhi/extension/io/cdc/siddhi-io-cdc/2.0.3/siddhi-io-cdc-2.0.3.jar"
+        },
+        "usages": [
+          {
+            "type": "BUNDLE",
+            "usedBy": "RUNTIME"
+          }
+        ],
+        "lookupRegex": "siddhi([_-])io([_-])cdc([_-])2.0.3.jar"
+      },
+      {
+        "name": "mssql-jdbc",
+        "version": "8.2.1.jre8",
+        "download": {
+          "autoDownloadable": true,
+          "url": "https://repo1.maven.org/maven2/com/microsoft/sqlserver/mssql-jdbc/8.2.1.jre8/mssql-jdbc-8.2.1.jre8.jar"
+        },
+        "usages": [
+          {
+            "type": "JAR",
+            "usedBy": "RUNTIME"
+          }
+        ],
+        "lookupRegex": "mssql([_-])jdbc([_-])8.2.1.jre8(.+).jar"
+      }
+    ]
+  },
+
+  "cdc-mongodb": {
+    "extension": {
+      "name": "cdc-mongodb",
+      "displayName": "Change Data Capture - Mongo DB",
+      "version": "2.0.3"
+    },
+    "dependencies": [
+      {
+        "name": "siddhi-io-cdc",
+        "version": "2.0.3",
+        "download": {
+          "autoDownloadable": true,
+          "url": "https://repo1.maven.org/maven2/io/siddhi/extension/io/cdc/siddhi-io-cdc/2.0.3/siddhi-io-cdc-2.0.3.jar"
+        },
+        "usages": [
+          {
+            "type": "BUNDLE",
+            "usedBy": "RUNTIME"
+          }
+        ],
+        "lookupRegex": "siddhi([_-])io([_-])cdc([_-])2.0.3.jar"
+      },
+      {
+        "name": "mongo-java-driver",
+        "version": "3.4.2",
+        "download": {
+          "autoDownloadable": true,
+          "url": "https://repo1.maven.org/maven2/org/mongodb/mongo-java-driver/3.4.2/mongo-java-driver-3.4.2.jar"
+        },
+        "usages": [
+          {
+            "type": "JAR",
+            "usedBy": "RUNTIME"
+          }
+        ],
+        "lookupRegex": "mongo([_-])java([_-])driver([_-])3.4.2(.+).jar"
```

Comment:
Won't there be `(.*)` instead of `(.+)` at the end, similar to the other ones we have changed above?